bridge/engine/ActiveMediaList.cpp

```diff
@@ -664,6 +664,81 @@ bool ActiveMediaList::makeUserMediaMapMessage(const size_t lastN,
     return true;
 }
 
+bool ActiveMediaList::makeBarbellUserMediaMapMessage(
+    const concurrency::MpmcHashmap32<size_t, EngineAudioStream*>& engineAudioStreams,
+    const concurrency::MpmcHashmap32<size_t, EngineVideoStream*>& engineVideoStreams,
+    utils::StringBuilder<1024>& outMessage)
+{
+    JsonObject umm(outMessage);
+    umm.addProperty("type", "user-media-map");
```

Comment:
I understand the utility but feels strange have a destructor that will add the final bracket and the data stay on the variable passed on constructor.

Also `JsonObject` is confusing. I guess we should consider a better name for it. Unfortunately I am terrible to find goof names :) but I have some ideas:`JsonObjectEncoder`, `JsonObjectWriter`...